### PR TITLE
mmsd: init at 1.12.1 with module

### PR DESCRIPTION
**Pull request by ju1m**
###### Description of changes

Package [Multimedia Messaging Service Daemon - The Next Generation](https://gitlab.com/kop316/mmsd) and add a NixOS service launched through the D-Bus by apps like Purism's [chatty](https://source.puri.sm/Librem5/chatty/-/blob/master/README.md#build-and-install-mmsd-tng).

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by AndersonTorres — APPROVED** [review]

**Review by RaitoBezarius — APPROVED** [review]
Could you add a release note for this? I think we are good to merge after that.

**Comment by ju1m** [comment]
- [X] Added a release note for `23.05`.

**Comment by RaitoBezarius** [comment]
Alas, more conflicts on release notes :'( ; please fix the conflicts and I think we are good for merge.

File: pkgs/tools/networking/mmsd-tng/default.nix
Comment (AndersonTorres) [inline pkgs/tools/networking/mmsd-tng/default.nix:49]:
```suggestion
    license = licenses.gpl2Plus;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Otherwise is OK

**Comment by ju1m** [comment]
- [X] Fix merge conflict on `rl-2305.section.md`
- [X] Fix `meta.license`

File: pkgs/tools/networking/mmsd-tng/default.nix
Comment (ju1m) [reply pkgs/tools/networking/mmsd-tng/default.nix]: Thanks

**Comment by RaitoBezarius** [comment]
So sorry it is taking a while, again merge conflicts, as now, we have two approvals, we can proceed just after the conflicts being fixed.

**Comment by ju1m** [comment]
- [X] Rebase to fix merge conflict with 08d651764f59

**Comment by RaitoBezarius** [comment]
> * [x]  Rebase to fix merge conflict with [08d6517](https://github.com/NixOS/nixpkgs/commit/08d651764f5955acd3800c65bff4934762dd2f41)

Unfortunately, still seeing conflicts :'( (so many PRs being merged, and we are now at 4.3k… PRs)

**Comment by ju1m** [comment]
- [X] Fix merge conflict in `rl-2305.section.md`

**Review by RaitoBezarius — APPROVED** [review]

**Pull request merged**
